### ISSUE TITLE
minor bug: text charset [junior job]

### ISSUE THREAD

**Issue by tushevorg** [owncloud/ios#368]
Text files that are not in Unicode (Windows-1251 for instance) are not displayed correctly.

It would be nice to have a default charset option somewhere in Settings.

**Comment by tushevorg** [comment]
Any updates?

**Comment by ggdiez** [comment]
Hello @tushevorg 

Yes, could be possible add this option in Settings and modify the charset of the UIWebView that manage the documents. For the moment we have not a plan to include this option. 

You are more than welcome to start contributing on the ownCloud project!

Before starting, we'd appreciate you sign the contribution agreement, http://owncloud.org/contribute/agreement/ and send it to @karlitschek

You may be found useful to read https://github.com/owncloud/ios/blob/develop/CONTRIBUTING.md

In case you want to contact us not only we are on github but also on the IRC, freenode #owncloud-ios

**Comment by tushevorg** [comment]
Thanks, @ggdiez , I will

**Comment by Jupirage** [comment]
>For the moment we have not a plan to include this option.
It's a pity. Without the ability of sharing texts between my pc and my iPhone, the whole idea of ownCloud loses its value.

**Comment by guruz** [comment]
Notepad++ should allow saving in UTF8 which I think is what iOS is using? (didn't try)

**Comment by Jupirage** [comment]
Thank you for the workaround. I'v just tried it and it's working (only I use Notepad2 rather than Notepad++)

**Comment by michaelstingl** [comment]
The ownCloud iOS team started to write a new iOS app from scratch. This version is deprecated and won't see major improvements.

Here you can find more information:
https://owncloud.org/news/working-brand-new-ios-app-sdk/
(Link to free TestFlight beta in the article)

Roadmap of new iOS app:
https://github.com/owncloud/ios-app/issues/94
